### PR TITLE
feat(tokenList): add tokenList rpcs/schemas

### PR DESCRIPTION
**Pull request by yqrashawn**
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/119)

### CODE REVIEW

File: packages/background/src/db-schema.js [hunk 1]
```diff
@@ -56,6 +56,16 @@ const schema = {
     builtin: {doc: "Indicating builtin network, shouldn't be deleted"},
     scanUrl: {doc: 'Url of block chain explorer'},
     selected: {doc: 'network selected by wallet'},
+    tokenList: {
+      doc: 'token list of this network',
+      ref: true,
+    },
+    token: {
+      doc: 'tokens from token list, dapp, and user of this network',
```
Comment (angelia-yuqi-personal) [inline packages/background/src/db-schema.js:64]: token from dapp and user may be account's data. why is `token` in. network here?

Comment (yqrashawn) [reply packages/background/src/db-schema.js:64]: token belongs to network/address/tokenList

wallet will only delete tokens if their network got removed, not address/tokenList

Comment (angelia-yuqi-personal) [reply packages/background/src/db-schema.js:64]: So tokens here contains token from tokenList/user adding/dapp adding, covers all the tokens displaying in ui. Is that right?

Comment (yqrashawn) [reply packages/background/src/db-schema.js:64]: yes, it's "token" as token in blockchain

**Review by angelia-yuqi-personal — APPROVED** [review]

**Pull request merged**